kim/field.py

```diff
@@ -442,6 +442,49 @@ class UserMapper(Mapper):
     marshal_pipeline = DecimalMarshalPipeline
     serialize_pipeline = DecimalSerializePipeline
 
+class FloatFieldOpts(FieldOpts):
```

Comment:
I think actually Decimal should inherit from Float. Could you do it that way round? Otherwise this looks great, thanks.